pkg/tools/ippool/ippool.go

```diff
@@ -230,6 +235,30 @@ func (tree *IPPool) PullP2PAddrs(exclude ...*IPPool) (srcNet, dstNet *net.IPNet,
 	return srcNet, dstNet, nil
 }
 
+// GetPrefixes returns the list of saved prefixes
+func (tree *IPPool) GetPrefixes() (prefixes []string) {
+	tree.lock.Lock()
+	clone := tree.clone()
+	tree.lock.Unlock()
+
+	if clone.root == nil {
+		return nil
+	}
+
+	it := iterator{
+		node: clone.root,
+	}
+	for it.node.Left != nil {
+		it.node = it.node.Left
+	}
+
+	for node := it.Next(); node != nil; node = it.Next() {
+		prefixes = append(prefixes, node.getPrefixes(clone.ipLength)...)
+	}
+
+	return
```

Comment:
I wonder why linter does not trigger at this line. 
Can we do not use return without values?

Reply:
Done